src/util/workerthread.cpp

```diff
@@ -59,6 +61,11 @@ void WorkerThread::run() {
             m_name.isEmpty() ? QString::number(threadNumber) : QString("%1 #%2").arg(m_name, QString::number(threadNumber));
     QThread::currentThread()->setObjectName(threadName);
```

Comment:
For the sake of consistency we should then remove the redundant `QThread::currentThread()->` prefix here. Seems to be a left over or I didn't notice back then.